external-plugins/rehearse/plugin/handler/handler.go

```diff
@@ -263,6 +266,43 @@ func (h *GitHubEventsHandler) handleRehearsalForPR(log *logrus.Entry, pr *github
 	}
 }
 
+func (h *GitHubEventsHandler) extractJobNamesFromComment(body string) []string {
+	if body == "" {
+		return nil
+	}
+	var jobNames []string
+	allStringSubmatch := rehearseCommentRe.FindAllStringSubmatch(body, -1)
+	for _, subMatches := range allStringSubmatch {
+		if len(subMatches) < 2 {
```

Comment:
IIUC it will happen in case of 

```
 /rehearse
```

since there will not be a submatch, no?

Reply:
I've experimented with the cases, it seems that even in the case where the command actually would be `^/rehearse$` the slice would be of len(2). I didn't found it that obvious when looking at the documentation so I'd say better be safe that sorry.

Even if this may never happen, I'd rather be keeping the guard. WDYT?